app/models.py

```diff
@@ -1352,10 +1352,13 @@ def check_code(self, cde):
 
 FIRST_CLASS = 'first'
 SECOND_CLASS = 'second'
-POSTAGE_TYPES = [FIRST_CLASS, SECOND_CLASS]
+EUROPE = 'Europe'
+REST_OF_WORLD = 'rest of world'
```

Comment:
This should change (again sorry) to:
```suggestion
EUROPE = 'europe'
REST_OF_WORLD = 'rest-of-world'
```
Based on @leohemsted’s comment here: https://github.com/alphagov/notifications-utils/pull/703#pullrequestreview-385420470

Reply:
Thanks, I've updated the PR to use those new values